project_transportonly/dimensions/subset/subsectors_by_sector.csv

```diff
@@ -0,0 +1,25 @@
+id,commercial_subsectors,industrial_subsectors,residential_subsectors,transportation_subsectors
```

Comment:
This file needs some work. The subsector IDs need to match the project's base dimension subsectors (like `Bus+Local+BEV_200`).

Reply:
industrial, commercial, etc., can be deleted.